dcos_test_utils/gce.py

```diff
@@ -0,0 +1,205 @@
+import logging
+
+from googleapiclient import discovery
+from googleapiclient.errors import HttpError
+from oauth2client.client import GoogleCredentials
+from retrying import retry
+
+from dcos_test_utils.helpers import Host
+
+log = logging.getLogger(__name__)
+
+
+class Instance:
+    def __init__(self, name, priv_ip, pub_ip):
+        self.name = name
+        self.private_ip = priv_ip
+        self.public_ip = pub_ip
+        self.host = Host(self.private_ip, self.public_ip)
```

Comment:
This should be a property. (if you change your self.private_ip you will want self.host to change as well)